src/helm/benchmark/huggingface_registration.py

```diff
@@ -19,6 +19,18 @@
 from helm.common.hierarchical_logger import hlog
 
 
+def get_default_huggingface_model_metadata(helm_model_name: str) -> ModelMetadata:
```

Comment:
1. Remove `huggingface` from the name.
2. Move this to `model_metadata_registry`.

Rationale: After #1996 we can also have non-Hugging Face private models that this should handle as well.